Assets/HoloToolkit/UX/Textures/KeyboardAtlas.spriteatlas

```diff
@@ -28,11 +28,11 @@ SpriteAtlas:
       enableTightPacking: 0
     packedHash:
       serializedVersion: 2
-      Hash: 59447b9d27cd467b975647bc4533e86e
+      Hash: b579d9a28ce9cc7d8109d453a20bac11
```

Comment:
This seems to update almost every time I open the project. Is there any way to get it to stay the same?

Reply:
Good question. I noticed the same thing.